doc/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -43,13 +56,13 @@ We will have to generate a YAML configuration file for configuring the 3D sensor
 
 
 YAML Configuration file (Depth Map)
------------------------------------
++++++++++++++++++++++++++++++++++++
 
 We will have to generate a YAML configuration file for configuring the 3D sensors. An `example file for processing depth images <https://github.com/ros-planning/panda_moveit_config/blob/master/config/sensors_kinect_depthmap.yaml>`_ can be found in the `panda_moveit_config repository <https://github.com/ros-planning/panda_moveit_config>`_ as well: ::
 
  sensors:
    - sensor_plugin: occupancy_map_monitor/DepthImageOctomapUpdater
-     image_topic: /head_mount_kinect/depth_registered/image_raw
```

Comment:
change in `ros-planning/panda_moveit_config`

Reply:
Can we keep this the default image_topic in panda_moveit_config?

Reply:
created [PR](https://github.com/ros-planning/panda_moveit_config/pull/4)